solver/jobs.go

```diff
@@ -350,7 +351,23 @@ func (jl *Solver) getState(e Edge) *state {
 	return st
 }
 
-func (jl *Solver) getEdge(e Edge) *edge {
+func (jl *Solver) getEdge(e Edge) (redge *edge) {
+	if debugScheduler {
```

Comment:
Shouldn't this be `redge.debug`?

Reply:
They are essentially the same, unless you are using the separate `BUILDKIT_SCHEDULER_DEBUG_STEPS` env var to specify an exact vertex name to log rather than just `BUILDKIT_SCHEDULER_DEBUG` which enables it for all vertexes. But in that case it should probably be checking `e.(*edge).debug` instead as opposed to `redge.debug`. But I wasn't really sure if any of this was worth thinking about since `BUILDKIT_SCHEDULER_DEBUG_STEPS` is extremely hard to use for this particular debugging use case.